Gemfile

```diff
@@ -3,9 +3,12 @@ source 'https://rubygems.org'
 # 64bit Amazon Linux 2016.09 v2.3.0 running Ruby 2.3 (Puma)
 ruby '2.3.1'
 
-gem 'puma'
-
+gem 'activerecord'
+gem 'activerecord-postgis-adapter'
```

Comment:
@eitoball can you confirm https://github.com/Safecast/safecastapi/blob/master/lib/tasks/db.rake#L1-L5 won't be a problem with the latest activerecord-postgis-adapter?

If we're gonna bring it in I'd like to avoid the sort of task patching we had to do there.